airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -350,6 +361,22 @@ def execute(self, context) -> Optional[str]:
                 "--project",
                 self.project_id,
             ]
+            if self.impersonation_chain:
+                if isinstance(self.impersonation_chain, str):
+                    impersonation_chain = self.impersonation_chain
```

Comment:
This variable name is confusing. The chain suggests it will be. multiple accounts, but you have one account. This variable (`impersonation_chain`) can also be confused with the `self.impersonation_chain`  field. What do you think to rename it to ``impersonation_account``?

Reply:
@mik-laj I wanted to change this to `impersonation_service_account` but then we don't need to check if we have more that one element in it, also I wanted to be consistent between operators that why I used `impersonation_chain`.

Reply:
@mik-laj Done

Reply:
The previous use of the impersonation chain looked fine for me. I just had my doubts about the name of the local variable. A local variable has never stored multiple accounts, but always only one account.

Reply:
👍

Reply:
@mik-laj I adjust my solution